omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -91,15 +91,17 @@
             move "#{install_dir}/etc/datadog-agent/datadog.yaml.example", "#{output_config_dir}/etc/datadog-agent"
             move "#{install_dir}/etc/datadog-agent/conf.d", "#{output_config_dir}/etc/datadog-agent", :force=>true
             unless heroku_target?
-              move "#{install_dir}/etc/datadog-agent/system-probe.yaml.example", "#{output_config_dir}/etc/datadog-agent"
+              if ENV.has_key?('SYSTEM_PROBE_BIN') and not ENV['SYSTEM_PROBE_BIN'].empty?
```

Comment:
Suggestion:
```suggestion
              if ENV.fetch("SYSTEM_PROBE_BIN", "").empty?
```

[doc](https://ruby-doc.org/core-2.7.0/Hash.html#method-i-fetch)

Would it be worth turning this condition into a function?

Reply:
As an aside, the conditionals on top of this (`if linux_target?` and `unless heroku_target?` seem to be yet another repetition of the condition that I referred to on the other review comment.